content/neuromorphic-computing/software/snn-frameworks/nestml/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "NESTML"
+type: neuromorphic-software
```

Comment:
Judging by the description, isn't this more like a (intermediate) language similar to NIR? In that case, I would classify it as a data tool more than a simulator.

Reply:
Thanks - that makes sense. I'll move it to data tools